database.go

```diff
@@ -990,6 +1018,17 @@ func (rp *RetentionPolicy) shardGroupByID(shardID uint64) *ShardGroup {
 	return nil
 }
 
+// dropSeries will delete all data with the seriesID
```

Comment:
My understanding is that is the only way to walk through to get the shards to delete the series.  Is there a faster way to isolate the shards I need?